packages/gatsby-theme-carbon/src/components/LeftNav/LeftNavTree.module.scss

```
@@ -7,17 +7,19 @@
 
 .container {
   height: 100%;
+  width: 100%;
+  position: absolute;
   margin: spacing.$spacing-10 0;
-  background-color: theme.$background;
+  background-color: theme.$text-inverse;
```

```suggestion
  background-color: theme.$layer-01;
```

Should probably be using a layer token. I think this is it?

---

@szinta brought up that `$layer` doesn’t map quite right for the themes — we need `white` in light theme but `#161616` in dark theme, which only `$text-inverse` seems to match.

She will follow the approach from regular left nav and add custom styles here.

---

used $background token for dark theme and added custom style overrides for g10 theme